src/reactA11yAnchorsRule.ts

```diff
@@ -245,6 +252,11 @@ class ReactA11yAnchorsRuleWalker extends ErrorTolerantWalker {
         } else if (root.kind === ts.SyntaxKind.JsxExpression) {
             const expression: ts.JsxExpression = <ts.JsxExpression>root;
             title += this.anchorText(expression.expression);
+        } else if (isChild && root.kind === ts.SyntaxKind.JsxSelfClosingElement) {
+            const jsxSelfClosingElement = <ts.JsxSelfClosingElement>root;
+            if (jsxSelfClosingElement.tagName.getText() !== 'img') {
```

Comment:
Did not find a good way to diff `html` jsx element than customised jsx element in this case.
So currently only check if children component is not img since `imageAlt` will check `altText ` with `anchorText `.
Passes all current test cases though.

Reply:
Yeah this seems fine for now. I don't know a better way than to directly get the text of the `tagName`...